pkg/proxy/iptables/proxier.go

```diff
@@ -1443,6 +1462,12 @@ func (proxier *Proxier) syncProxyRules() {
 	// Ref: https://github.com/kubernetes/kubernetes/issues/74839
 	// Ref: https://github.com/kubernetes/kubernetes/issues/117924
 	if !proxier.conntrackTCPLiberal {
+		proxier.filterAccountingRules.Write(
```

Comment:
Is there some reason I'm missing for `proxier.filterAccountRules`? ISTM that you should just use `proxier.filterRules` here.

Actually, you don't even need a separate rule; you can just insert the nfacct expression between the "--ctstate INVALID" and the "-j DROP" of the existing rule. (I think.)